p2p/src/config.rs

```diff
@@ -47,17 +47,15 @@ pub enum NodeType {
     Inactive,
```

Comment:
It would be great if we could remove this

Reply:
It's used in the tests, so I added the `cfg` attribute.